.vscode/launch.json

```diff
@@ -42,7 +59,21 @@
           "remoteRoot": "${workspaceFolder}"
         }
       ],
-      "justMyCode": false
+      "justMyCode": false,
+      "presentation": {
+        "hidden": true,
+      }
+    }
+  ],
+  "compounds": [
+    {
+      "name": "Debug selected launch target",
+      "configurations": ["C/C++ debugger", "Python debugger"],
+      //"preLaunchTask": "DebuggerPreLaunchTask",
```

Comment:
what's up with this line? is this something that we might want to uncomment or just leftover that wasn't cleaned?

Reply:
As i've explained in one of my last comments, right now I see no way of automatically activating the conda env. This implies having to manually activate the environment when you open vscode. This is left there just in case we find a solution.